docs/.wordlist.txt

```diff
@@ -47,3 +47,22 @@ YAML
 STSStack
 sshuttle
 vpn
+behavior
+HH
+YYYY
+ss
+SQLAlchemy
+Caracal
+Xena
+fbcab
+ccb
+aac
+de
+acf
+ef
+ec
```

Comment:
I wonder if the spell checker should automatically detect hex strings. :thinking: 

Either way, the solution here for both readability and avoiding spellcheck issues is to wrap the migration identifiers in code blocks, like:

```
       - ``16f1fbcab42b_resolve_shadow_table_diffs``
```